libcomposefs/meson.build

```diff
@@ -0,0 +1,44 @@
+composefs_hash_cflags = ['-DUSE_OBSTACK=0', '-DTESTING=0', '-DUSE_DIFF_HASH=0']
+
+source_files = files([
+  'bitrotate.h',
+  'erofs_fs.h',
+  'erofs_fs_wrapper.h',
+  'hash.c',
+  'hash.h',
+  'lcfs-internal.h',
+  'lcfs-erofs.h',
+  'lcfs-erofs-internal.h',
+  'lcfs-fsverity.c',
+  'lcfs-fsverity.h',
+  'lcfs-writer-erofs.c',
+  'lcfs-writer.c',
+  'lcfs-writer.h',
+  'lcfs-utils.h',
+  'lcfs-mount.c',
+  'lcfs-mount.h',
+  'xalloc-oversized.h',
+])
+
+libcomposefs = shared_library('composefs', 
+  source_files,
+  c_args : composefs_hash_cflags + hidden_visibility_cflags,
+  dependencies : libcrypto_dep,
+  version : meson.project_version(), # FIXME: or libversion?
```

Comment:
How does this relate to https://github.com/containers/composefs/pull/300/files#diff-30d8f6be6320feeacf686be94f48c70869b52630e01ea625f0f15adc0d57c3e4R16 ?

Reply:
I saw that you have different versions for the project itself and the libcomposefs.so links. The pkgconfig file also includes a version field. Which version should go where? :)

Reply:
I've changed it to match what autotools produces (project version 1.0.4 in composefs.pc, libversion 1.2.0 for the symlinks).